src/commands/tag-image.yml

```diff
@@ -0,0 +1,25 @@
+description: Add a tag to an existing published image
+
+parameters:
+  repo:
+    type: string
+    description: Name of an Amazon ECR repository
+
+  source-tag:
+    type: string
```

Comment:
I would wager that images pushed without a `latest` tag will most commonly be tagged with the git sha. Therefore, I would propose that this parameter default to `<< pipeline.git.revision >>`.

Reply:
I tried to keep it inline with the other command that push the image with the `latest` tag